scripts/drone/events/release.star

```diff
@@ -42,6 +42,7 @@ load(
 )
 
 ver_mode = "release"
+semver_regex = r"^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
Spicy regex. 🌶️

Reply:
it's the official semver regex!

Reply:
Whoever had to build this was hating life for a few days.